src/pystack/_pystack/process.cpp

```diff
@@ -535,6 +535,20 @@ AbstractProcessManager::Pid() const
 {
     return d_pid;
 }
+
+remote_addr_t
+AbstractProcessManager::getAddressFromCache(const std::string& symbol) const
+{
+    return d_symbol_cache[symbol];
+}
+
+void
+AbstractProcessManager::registerAddressInCache(const std::string& symbol, remote_addr_t address) const
+{
+    d_symbol_cache[symbol] = address;
```

Comment:
Er, I think you meant to do
```suggestion
    d_type_cache[symbol] = address;
```